backend/src/auth/auth.controller.ts

```diff
@@ -72,22 +72,22 @@ export class AuthController {
   }
 
   @Post('logout')
-  async logout(
+  async logout (
     @Req() req: Request,
-    @Session() session: UserSession,
-    @Res({ passthrough: true }) res: Response,
+      @Session() session: UserSession,
+      @Res({ passthrough: true }) res: Response
   ): Promise<void> {
     res.clearCookie(this.config.get('session.name'))
     session.destroy(() =>
-      res.status(HttpStatus.OK).json({ message: 'Logged out' }),
+      res.status(HttpStatus.OK).json({ message: 'Logged out' })
     )
   }
 
   @Get('whoami')
-  async whoami(@Req() req: Request, @Res() res: Response): Promise<void> {
+  async whoami (@Req() req: Request, @Res() res: Response): Promise<void> {
     const user = req.session.user
     res
       .status(HttpStatus.OK)
-      .json(user ? { id: user.id, email: user.email } : null)
+      .json((user != null) ? { id: user.id, email: user.email } : null)
```

Comment:
```suggestion
      .json((user !== null) ? { id: user.id, email: user.email } : null)
```

Seems like `ts-standard` doesn't use strict equality.

Reply:
that's actually me - I wanted to cover both `null` and `undefined`, hence the deliberate use of `!=`. But I see this is causing confusion, so I'll rework to cover the `undefined` case too.